src/user-profile.js

```diff
@@ -189,15 +190,25 @@ const Profile = {
     }
 
     // Create a user config if one doesn't exist.
-    const userConfigPath = path.join(this.configDir, 'local.js');
-    if (!fs.existsSync(userConfigPath)) {
-      fs.writeFileSync(
-        userConfigPath, '\'use strict\';\n\nmodule.exports = {\n};');
+    const oldUserConfigPath = path.join(this.configDir, 'local.js');
+    const oldLocalConfigPath = path.join(this.gatewayDir, 'config', 'local.js');
+    const userConfigPath = path.join(this.configDir, 'local.json');
+
+    if (fs.existsSync(oldUserConfigPath)) {
+      const oldConfig = config.util.parseFile(oldUserConfigPath);
+      fs.writeFileSync(userConfigPath, JSON.stringify(oldConfig, null, 2));
```

Comment:
This can overwrite `userConfigPath` if the user happens to have one already. I think it would be better to only copy over the old config if userConfigPath doesn't exist and then fall back to `'{\n}'`.

Reply:
Nice catch. Fixed.